doc/src/build/move/build-test.md

```diff
@@ -0,0 +1,344 @@
+---
+title: Build and Test the Sui Move Package
+---
+
+## Building a package
+
+Ensure you are in the `my_move_package` directory that contains your package, and then use the following command to build it:
+
+``` shell
+$ sui move build
+```
+
+A successful build returns a response similar to the following:
+
+```shell
+Build Successful
+Artifacts path: "./build"
+```
+
+Now that we have designed our asset and its accessor functions, let us

```

Comment:
I am not clear on how the asset and its accessor functions were designed by running the build comand

Reply:
> I am not clear on how the asset and its accessor functions were designed by running the build comand

Do add that to your friction log and associated issue for editing these contents. Delving into the issue here is outside the scope of this PR.